tf_quant_finance/black_scholes/approximations/american_option.py

```diff
@@ -20,8 +20,10 @@
 from tf_quant_finance.black_scholes import vanilla_prices
 from tf_quant_finance.math import gradient
 from tf_quant_finance.math.root_search import newton as root_finder_newton
+from tensorflow.python.util import deprecation
 
 
+@deprecation.deprecated_args(None, "continuous_dividends is deprecated. Use dividend_rates instead", "continuous_dividends")
 def adesi_whaley(*,
                  volatilities,
```

Comment:
Thank you for cleaning up all methods. Could you please also modify Heston EU option pricer:
https://github.com/google/tf-quant-finance/blob/d99c50258cfcd12346f439d3bef1e733867cc707/tf_quant_finance/models/heston/approximations/european_option.py#L37

Reply:
Sure, I updated the Heston EU option pricing model as well.  Also, there was one reference to `continuous_dividends` in the Local Volatility Model, where it called the Black Scholes option pricer, so I updated that as well.